assets/src/blocks/Counter/CounterEditor.js

```diff
@@ -55,8 +55,9 @@ export const CounterEditor = ({setAttributes, attributes, isSelected}) => {
               onChange={toAttribute('text')}
             />
           </div>
-          <div className="sidebar-blocks-help">
-            {__('These placeholders can be used:', 'planet4-blocks-backend')}{' '}<code>%completed%</code>, <code>%target%</code>, <code>%remaining%</code>
+          <div className="components-base-control__help">
```

Comment:
Just a doubt from my side.. why did you have change the classname here?

Reply:
Thanks for the review @GP-Dan-Tovbein 
As I've mentioned in the PR description: "the class name of the last text div was changed to follow the design of the rest of the blocks."
I can revert this change if you consider it unnecessary.

Reply:
Oh sorry, I missed that part 🙏